src/configurations/destinations/mp/db-config.json

```diff
@@ -88,6 +89,7 @@
         "trackCategorizedPages",
         "trackNamedPages",
         "sourceName",
+        "sessionReplayPercentage",
```

Comment:
Isn't this feature specific only to web device ?

Reply:
should have been under defaultConfig.web

Reply:
updated